extensions/oci/eslint.config.js

```diff
@@ -3,4 +3,9 @@ const raycastConfig = require("@raycast/eslint-config");
 
 module.exports = defineConfig([
   ...raycastConfig,
+  {
+    rules: {
+      "@raycast/prefer-title-case": "off"
```

Comment:
Hey @xmok, how come we need this? Am I missing a line change here that requires it? 😅

Reply:
Yep, that makes sense, thank you!